src/common/interfaces.ts

```diff
@@ -203,6 +203,8 @@ export interface LineGraphType {
   width: number;
   height: number;
   data: Array<DateValueType>;
+  yAxisUnit?: string;
+  xAxisUnit?: string;
```

Comment:
isn't the xAxis unit always datetime?

Reply:
Yep, that's true. However I would leave the name of the unit flexible as a `string`, so that we're not limiting ourselves to one particular wording such as "Messdatum". Perhaps at some point we wanna rename it e.g. "Datum"...